src/styles/utils/variables.scss

```diff
@@ -0,0 +1,22 @@
+$roboto-font-family: 'Roboto', sans-serif;
+$font-weight-medium: 500;
+$font-weight-regular: 400;
+$font-size-small: 10px;
+$font-size-medium: 12px;
+$font-size-large: 16px;
+$line-height-small: 14px;
+$line-height-medium: 18px;
+$line-height-large: 18px;
+$font-size-bold: 14px;
+$line-height-bold: 16px;
+$font-weight-bold: 700;
+$font-style-normal: normal;
+$color-bacground: white;
+$color-bacground: white;
```

Comment:
There is a duplicate variable `$color-bacground` with a typo. It should be `$color-background`. Remove the duplicate and correct the typo.